net/lldpd/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+PortGroup             github 1.0
+
+github.setup          lldpd lldpd 1.0.19
+revision              0
+categories            net
+maintainers           {@akkornel stanford.edu:akkornel} openmaintainer
+license               ISC
+
+description           DESC
+long_description      ${description}, LONGDESC
```

Comment:
You can use the descriptions from the official website.

```
description           LLDP allows you to know exactly on which port is a server \
                      \(and reciprocally\).

long_description      ${description} LLDP is an industry standard protocol designed to \
                      supplant proprietary Link-Layer protocols such as EDP or CDP. The \
                      goal of LLDP is to provide an inter-vendor compatible mechanism to \
                      deliver Link-Layer notifications to adjacent network devices.

Reply:
Blargh, I completely missed that.  Re-pushing now…